data/_models/shared/Agent.csv

```diff
@@ -1,4 +1,6 @@
 tableName,tableExtends,columnName,columnType,key,required,refSchema,refTable,refLink,refBack,validation,semantics,description,profiles
-Agent,,,,,,,,,,,"http://xmlns.com/foaf/0.1/Agent","An agent (eg. person, group, software or physical artifact).","DCAT,Beacon v2,DataCatalogueFlat"
+Agent,,,,,,,,,,,"http://xmlns.com/foaf/0.1/Agent,http://www.w3.org/2006/vcard/ns#Agent","An agent (eg. person, group, software or physical artifact).","DCAT,Beacon v2,DataCatalogueFlat"
 Agent,,name,string,1,TRUE,,,,,,"http://xmlns.com/foaf/0.1/name,http://purl.org/dc/terms/identifier",A name and identifier for this agent.,"DCAT,Beacon v2,DataCatalogueFlat"
 Agent,,logo,hyperlink,,,,,,,,"http://xmlns.com/foaf/0.1/logo",A logo representing some thing.,"DCAT,Beacon v2,DataCatalogueFlat"
+Agent,,url,hyperlink,,,,,,,,"http://www.w3.org/2006/vcard/ns#url",An URL representing some thing.,"Beacon v2,DataCatalogueFlat"
```

Comment:
I am still a bit concerned about the agent as that is an alien entity in our model atm and I don't like it. 

I think we should have proper hierarchy, which might be agent, organisational unit extends agent and contact person extends agent.

We previously used to have 'organisation' as 'resource' so it had a unique name matching very will this model.
However, we also noticed that there are actually many agents usually not overlapping with the ROR but being a subunit of those. Anyway, sorry for the rant, we need to have a careful think on this.